packages/tse-backend/src/modules/exchange-operation/order.service.ts

```diff
@@ -17,7 +17,9 @@ export class OrderService {
 
   async persistOrderActivity(command: ExchangeOperationCommand): Promise<void> {
     const { orderEntityId, status, orderId, details } = command;
-    const order = await this.repository.findById(orderEntityId);
+    const order = orderEntityId
+      ? await this.repository.findById(orderEntityId)
+      : await this.repository.findByOrderId(orderId);
```

Comment:
What's the difference between `orderEntityId` and `orderId`?

Reply:
`orderEntityId` is `id` from `packages/tse-backend/src/common/entities/order.entity.ts`
`orderId` is `result.id` from `packages/tse-backend/src/modules/exchange-trade/exchange-trade.service.ts`:
```
const result = await exchangeInstance.createOrder(
        command.symbol,
        MarketOrderType.MARKET_ORDER,
        command.side,
        command.amount,
      );
      this.logger.log(
        `Market trade executed successfully: ${JSON.stringify(result)}`,
      );
      await this.saveExchangeOperation(
        savedData.id,
        OrderStatus.EXECUTED,
        result.id,
        result,
      );
```

I will rename orderId into orderExternalId to clarify.

Reply:
fix: b51ceb05